opensoundscape/annotations.py

```diff
@@ -610,8 +610,28 @@ def one_hot_labels_like(
 
         # the clip_df should have ['file','start_time','end_time'] as the index
         clip_df[classes] = float("nan")  # add columns for each class
-
-        for file, start, end in clip_df.index:
+        output_df = clip_df.copy()
+
+        for class_name in classes:
+            # set all the values to 0 for clip_df. We will add the 1s in the next step
+            output_df[class_name] = 0
```

Comment:
small thing but can just initialize to 0 in L612 unless there's a reason we might want nans if this loop is interrupted

Reply:
Made change